src/utils/Api.js

```diff
@@ -3,7 +3,7 @@ import getLanguage from './getLanguage.js';
 
 const host = `${window.location.protocol}//${window.location.host}`;
 
-const base = 'https://phantom-be-codebandits-staging.herokuapp.com/api/v1';
+const base = 'https://phantom-codebandits.herokuapp.com/api/v1';
 const axiosBase = axios.create({
   baseURL: base,
```

Comment:
remember to change the URL back to its initial value after the PR on the backend Repo is merged